pytorch_run.sh

```diff
@@ -68,6 +66,12 @@ if [ $stage -le 2 ]; then
   frame_shift=0.01
   awk -v frame_shift=$frame_shift '{print $1, $2*frame_shift;}' data/train/utt2num_frames > data/train/reco2dur
 
+
+  if [[ -f "./RIRS_NOISES" ]]; then
+      echo "ERROR: RIRS_NOISES noise dataset directory is not setup."
+      exit 1
+  fi
+
```

Comment:
Can we have this check at the beginning of the script, say line 31?
This will help user catch this error before downloading & preparing vox corpora

Reply:
Sure. I will reflect the things you mentioned. Thanks!